src/Interpreters/ChineseTokenizer.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "config.h"
+
+#if USE_CPPJIEBA
```

Comment:
Added `USE_CPPJIEBA` all over the place. Note that these are defined by "config.h", so we need to include that whenever we check `USE_CPPJIEBA`.